templates/index/index.html

```diff
@@ -207,168 +183,237 @@
-        transform: scale(1) translateY(10%);
-        animation: none;
-        opacity: 0.1;
+    @media (min-width: 1140px) {
+      .style-title {
+        font-size: 2.5rem;
+      }
+      .style-nav1-text {
+        font-size: 1.125rem;
+      }
+      .style-hero-links {
+        font-size: 1.25rem;
+      }
+      .style-card-header {
+        font-size: 1.75rem;
+      }
+      .style-button {
+        font-size: 0.875rem;
       }
     }
 
-    @media (max-width: 960px) {
-      .index-stats {
-        font-size: 0.9rem;
+    .style-badge {
+      display: inline-block;
+      white-space: nowrap;
+      height: 1rem;
+      text-align: center;
+      border-radius: 10.5px;
+      width: 1.3rem;
+    }
+
+    .style-badge .style-status-tag-text {
+      position: relative;
+      top: 50%;
+      transform: translateY(-50%);
+    }
+
+    .style-bg-good {
+      background-color: var(--success);
+    }
+
+    .style-bg-neutral-1 {
+      background-color: var(--light);
+    }
+
+    .style-bg-neutral-2 {
+      background-color: var(--dark);
+    }
+
+    .style-bg-bad {
+      background-color: var(--danger);
+    }
+
+    @media (max-width: 639px) {
+      .style-badge.style-badge-single-char {
+        width: 1rem;
+        padding: 0;
+        border-radius: 50%;
       }
+    }
 
-      .index-stats h5,
-      .card-title {
-        font-size: 1rem !important;
+    @media (min-width: 640px) {
+      .style-badge {
+        width: 5rem;
+      }
+      .style-badge-nfs {
+        width: 7rem;
       }
     }
 
-    [v-cloak] {
-      visibility: hidden;
+    .style-grid {
+      display: grid;
+      white-space: nowrap;
+      color: var(--font-color);
+      overflow: scroll;
+      overflow-x: auto;
+      overflow-y: overlay;
     }
 
-    .responsive-border-right {
-      border-right-color: rgb(222, 226, 230);
-      border-right-style: solid;
-      border-right-width: 1px;
+    .style-grid .style-grid-header {
+      background-color: var(--bg-table-header);
+      position: sticky;
+      top: 0;
+      z-index: 1;
     }
 
-    @media (max-width: 767px) {
-      .responsive-border-right-l {
-        border: hidden;
-      }
+    .style-grid .style-grid-header,
+    .style-grid .style-grid-cell {
+      display: flex;
+      align-items: center;
+    }
+
+    .style-grid .style-grid-cell {
+      border-bottom: 1px solid var(--border-color);
+    }
+
+    .style-grid div[role="row"]:last-of-type .style-grid-cell {
+      border: 0;
+    }
+
+    .style-grid::-webkit-scrollbar {
+      width: 5px;
+      height: 5px;
+    }
+    .style-grid::-webkit-scrollbar-thumb {
+      -webkit-border-radius: 10px;
+      border-radius: 10px;
+      background-color: var(--table-scroll);
+      opacity: 0;
+    }
+    .style-grid:hover::-webkit-scrollbar-thumb {
+      background-color: var(--table-scroll-hover);
     }
   </style>
+
+  {{ template "hero_css" }}
+  {{ template "postGenesis_css" }}
 {{ end }}
 
 {{ define "content" }}
```

Comment:
I am sorry, I am not sure where to put this conversation, but there is now a big gap between the latest epochs/slots and the footer:
![image](https://user-images.githubusercontent.com/114066135/210772199-df16fd54-ec2a-40ac-b43b-60dfea25757b.png)

Reply:
The height of the main content in a view changes based on the height of the browser window, probably to make sure the footer is not visible without scrolling but I'm not sure. Definitely wonky behavior but won't change for now.